config/jobs/dependency-watchdog/dependency-watchdog-kind-unit-tests.yaml

```diff
@@ -0,0 +1,77 @@
+presubmits:
+  gardener/dependency-watchdog:
+    - name: pull-dependency-watchdog-kind-unit-tests
+      cluster: gardener-prow-build
+      always_run: true
+      skip_branches:
+        - release-v\d+.\d+ # don't run on release branches for now (add a job per branch later)
+      decorate: true
+      decoration_config:
+        timeout: 60m
+        grace_period: 15m
+      labels:
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
+      annotations:
+        description: Runs KIND cluster based unit tests for dependency watchdog developments in pull requests
+        fork-per-release: "true"
```

Comment:
please remove `fork-per-release` annotation. The `job-forker` is enabled for g/g only atm.

Reply:
done